hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/CryptoCreateValidator.java

```diff
@@ -100,7 +100,7 @@ public boolean tooManyAutoAssociations(
             @NonNull final TokensConfig tokensConfig) {
         return (entitiesConfig.limitTokenAssociations() && n > tokensConfig.maxPerAccount())
                 || n > ledgerConfig.maxAutoAssociations()
-                || (n < 0 && !entitiesConfig.unlimitedAutoAssociationsEnabled())
```

Comment:
Please update the Javadoc of this method. We are not only checking the upper bound.

Reply:
Javadoc updated to reflect the changes